eslint-plugin-javascript/src/rules/use-app-alternatives.ts

```diff
@@ -0,0 +1,103 @@
+import { getFullFunctionName, parseOptions, PLUGIN_DOCS_URL } from '../utils';
+import type { Rule } from 'eslint';
+
+interface AppAlternative {
+  alternativeTo: string;
+  appFunction: string;
+}
+
+interface RuleOptions {
+  alternatives: AppAlternative[];
+}
+
+const defaultRuleOptions: RuleOptions = {
```

Comment:
Remove these defaults and add them to the `eslint-config` instead.

Reply:
We will be adding the eslint-configs changes after we have published the new plugins on npm